src/config.h

```diff
@@ -59,6 +56,13 @@
 
 ////#define FAKE_VESC_ENABLED
 
+#define WIFI_ENABLED
```

Comment:
Wouldn't it be better to allow setting these values via App? Many guys don't build the firmware on their own.